pkg/filebacked/file.go

```diff
@@ -1,7 +1,6 @@
 /*
 File backing for collections.
 File format:
-   Length: 8 (uint64)
```

Comment:
Not related but noticed.  The file format no longer includes the `Length` header.